packages/primitives-core/package.json

```diff
@@ -13,8 +13,8 @@
     "css-to-react-native": "^2.2.1"
   },
   "peerDependencies": {
-    "@emotion/core": ">=10.0.0-beta",
-    "react": ">=16.3.0"
+    "@emotion/core": "10.x.x || 10.0.0-beta.x",
```

Comment:
This change to the `@emotion/core` peerDep version should be reverted

Reply:
Could you tell me the reason?
I'm just curious.

Reply:
@mitchellhamilton Do I need to change peerDeps of `emotion-server` too? It uses
```json
  "peerDependencies": {
    "emotion": "10.x.x || 10.0.0-beta.x"
  },
```